ui/src/global-constants.ts

```diff
@@ -4,4 +4,6 @@ export const DEFAULT_ROUTING_PORT = "default";
 export const DEFAULT_NODE_SIZE = { width: 150, height: 25 };
 export const ALLOWED_WORKFLOW_FILE_EXTENSIONS = ".json, .yaml, .yml";
 export const ALLOWED_PROJECT_IMPORT_EXTENSIONS = ".zip";
+export const ALLOWED_ASSET_IMPORT_EXTENSIONS =
+  ".csv, .geojson, .gml, .json, .sql, .tsv";
```

Comment:
.sql doesn't exist. Sorry when I explained the allowed asset imports I should have caught that. We don't need to do anything with SQL since that functionality is fully action/engine related.